projects/aws/eks-anywhere-build-tooling/Makefile

```diff
@@ -1,65 +1,33 @@
 BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
-BUILD_LIB=$(BASE_DIRECTORY)/build/lib
 GIT_TAG?=$(shell cat GIT_TAG)
-BRANCH_NAME?=main
-LATEST_TAG?=latest
-ifneq ("$(BRANCH_NAME)","main")
-	LATEST_TAG=$(BRANCH_NAME)
-endif
 
-AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
-AWS_REGION=us-west-2
+REPO=aws
```

Comment:
Should this be the other way around? Repo owner = aws, repo = eks-anywhere-build-tooling